lms/djangoapps/courseware/views/views.py

```diff
@@ -561,11 +561,6 @@ def register_user_access_warning_messages(request, course):
                             enroll_link_end=HTML('</button>')
                         )
                     )
-                else:
-                    PageLevelMessages.register_warning_message(
-                        request,
-                        Text(_('You must be enrolled in the course to see course content.'))
-                    )
```

Comment:
Any particular reason for removing this text? This should be shown when the course is set to `invitation_only`. So you can keep this else statement as is.

Reply:
ok sure thing. Yes, they should match.